src/assets/styles/overrides.scss

```diff
@@ -144,5 +144,5 @@
   margin-top: 0 !important;
   border-right: 10px solid transparent !important;
   border-left: 10px solid transparent !important;
-  border-top: 8px solid $app-blue !important;
+  border-top: 8px solid RGBA(73,80,87,0.95) !important;
```

Comment:
Is this to correct the colour of the arrow?
I also noticed all the tooltips are pointing to the left... i'm curious if we changed something globally recently?

![Screen Shot 2021-09-09 at 1 34 21 PM](https://user-images.githubusercontent.com/53542131/132758949-ba7067a6-4c17-4648-a9b3-c5d292ae0d2b.png)

Reply:
Might have been me... can't remember!